#MM-34893 removed a section about Admin Advisor as the feature is now disabled (#4549)
mattermost/docs: source/overview/compliance.rst

Clicking **Contact us** in the Mattermost Admin Advisor will send some information to us. This may include the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number. This information is used to contact you as requested and to help us better understand your needs.

.. note::

>     Mattermost `Admin Advisor notices are disabled <https://docs.mattermost.com/administration/notices.html#admin-advisor-notices>`__ in Mattermost Server from v5.35.

Reviewer: ```suggestion
    `Mattermost Admin Advisor notices are disabled <https://docs.mattermost.com/administration/notices.html#admin-advisor-notices>`__ in v5.35 and later.
```
